go/database/mpt/forest.go

```diff
@@ -383,6 +383,24 @@ func (s *Forest) SetValue(rootRef *NodeReference, addr common.Address, key commo
 	return newRoot, err
 }
 
+func (s *Forest) HasEmptyStorage(rootRef *NodeReference, addr common.Address) (isEmpty bool, err error) {
+	v := &lambdaVisitor{
```

Comment:
there is a constructor for this visitor: https://github.com/Fantom-foundation/Carmen/blob/main/go/database/mpt/visitor.go#L71